src/poller.hpp

```diff
@@ -48,6 +48,13 @@
 #include "poll.hpp"
 #elif defined ZMQ_USE_SELECT
 #include "select.hpp"
+#elif defined ZMQ_HAVE_GNU
+#define ZMQ_USE_SELECT
+#include "select.hpp"
+#if 0
```

Comment:
Hello,

Luca Boccassi, on Sat 16 Jan 2016 05:13:05 -0800, wrote:
> > @@ -48,6 +48,13 @@
> >  #include "poll.hpp"
> >  #elif defined ZMQ_USE_SELECT
> >  #include "select.hpp"
> > +#elif defined ZMQ_HAVE_GNU
> > +#define ZMQ_USE_SELECT
> > +#include "select.hpp"
> > +#if 0
> 
> Any reason to keep this #if 0 ?

I don't see any.  I don't know if Svante tried to use poll, but at least
select should be working fine.

Samuel